### ISSUE TITLE
Potential Issues with Jump Range Calculations in edshipyard.export

### ISSUE THREAD

**Issue by Rixxan** [EDCD/EDMarketConnector#2229]
**Describe the bug**
EDMC may be calculating and outputting laden and unladen ranges incorrectly compared to the values provided in-game. It is unclear why these are the way that they are, and should be investigated to determine where the fault is being introduced. (My guess is with the multiplier calculation)

**To Reproduce**
* Run from command-line `python edmc.py --loglevel DEBUG -e test2.txt` with any ship/module combination known to EDMC
* Compare to in-game displayed values

**Expected behavior**

**Screenshots**
![image](https://github.com/EDCD/EDMarketConnector/assets/26337384/2d19d132-b028-4ccd-88be-53b4063d2cd9)

**Comment by Rixxan** [comment]
OK so first detail: laden and unladedn per EDMC are not the same as the game's. In EDMC laden == max fuel load + max cargo, where unladen == max fuel load with no cargo. This does not take into account various fuel states beyond maximum.

**Comment by Athanasius** [comment]
Indeed, *this* EDMC code is never going to take into account "actual current fuel load", because that would be a Status.json thing and it's only taking into account the pure loadout.

There's also the wrinkle, and I forget what the game does wrt, about the Active Fuel Reservoir fuel mass.

**Comment by Rixxan** [comment]
Given that this is "as intended", closing because I didn't understand how the data flowed. Thank you Athan for the insight :)
